packages/openapi-codegen-client-fetch/src/parts/ActionFunc.test.ts

```diff
@@ -162,8 +162,10 @@ test('simple test', () => {
       addQueryParam(url, 'offset', params.offset);
       addQueryParam(url, 'limit', params.limit);
 
+      const headers = {...COMMON_HEADERS}
```

Comment:
This line should end with `;`, otherwise some linter checks for generated code may not pass.